src/server/_utils/pricing.ts

```
@@ -0,0 +1,52 @@
+import { env } from "~/env";
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)